### ISSUE TITLE
Compiling error E0283

### ISSUE THREAD

**Issue by lebensterben** [lotabout/skim#188]
I was trying to install `skim` via `cargo` and the following error occurred.

```
   Compiling skim v0.6.7
     Running `rustc --edition=2018 --crate-name skim /home/lucius/.cargo/registry/src/github.com-1ecc6299db9ec823/skim-0.6.7/src/lib.rs --color always --crate-type lib --emit=dep-info,link -C opt-level=3 --cfg 'feature="default"' -C metadata=a04a50de2ae59014 -C extra-filename=-a04a50de2ae59014 --out-dir /tmp/cargo-install8CeDWS/release/deps -L dependency=/tmp/cargo-install8CeDWS/release/deps --extern bitflags=/tmp/cargo-install8CeDWS/release/deps/libbitflags-7da479843b005ba4.rlib --extern chrono=/tmp/cargo-install8CeDWS/release/deps/libchrono-cf861bfc2931365a.rlib --extern clap=/tmp/cargo-install8CeDWS/release/deps/libclap-3884add869b87984.rlib --extern derive_builder=/tmp/cargo-install8CeDWS/release/deps/libderive_builder-807bbaf018e8f267.so --extern env_logger=/tmp/cargo-install8CeDWS/release/deps/libenv_logger-4955433a86104d7c.rlib --extern fuzzy_matcher=/tmp/cargo-install8CeDWS/release/deps/libfuzzy_matcher-b9f152d64ff9e136.rlib --extern lazy_static=/tmp/cargo-install8CeDWS/release/deps/liblazy_static-56fd04421d7a44b1.rlib --extern log=/tmp/cargo-install8CeDWS/release/deps/liblog-823d679605fab846.rlib --extern nix=/tmp/cargo-install8CeDWS/release/deps/libnix-a800fcb68f6367fa.rlib --extern rayon=/tmp/cargo-install8CeDWS/release/deps/librayon-367d697d4a5534be.rlib --extern regex=/tmp/cargo-install8CeDWS/release/deps/libregex-d05f5024a307020a.rlib --extern shlex=/tmp/cargo-install8CeDWS/release/deps/libshlex-800fb40da39b7401.rlib --extern time=/tmp/cargo-install8CeDWS/release/deps/libtime-6b14c6221822b10c.rlib --extern timer=/tmp/cargo-install8CeDWS/release/deps/libtimer-b2432a3b9e64419a.rlib --extern tuikit=/tmp/cargo-install8CeDWS/release/deps/libtuikit-d0145bc165de8889.rlib --extern unicode_width=/tmp/cargo-install8CeDWS/release/deps/libunicode_width-138429304f8bb71a.rlib --extern vte=/tmp/cargo-install8CeDWS/release/deps/libvte-c99aeea6a2fee451.rlib --cap-lints allow`
error[E0283]: type annotations required: cannot resolve `_: std::convert::Into<tuikit::container::Size>`
   --> /home/lucius/.cargo/registry/src/github.com-1ecc6299db9ec823/skim-0.6.7/src/model.rs:567:14
    |
567 |             .basis(if self.inline_info { 0 } else { 1 }.into())
    |              ^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0283`.
error: failed to compile `skim v0.6.7`, intermediate artifacts can be found at `/tmp/cargo-install8CeDWS`
```

**Comment by lotabout** [comment]
@lebensterben Could you please share which commit of skim are you using? as well as the version of rust?

**Comment by h-michael** [comment]
@lotabout Same thing occurred.
That happens when `cargo install skim` from crates.io.

```
~> rustup show
Default host: x86_64-unknown-linux-gnu

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu (default)
beta-x86_64-unknown-linux-gnu
nightly-2018-11-03-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
master
1.15.1-x86_64-unknown-linux-gnu
1.30.1-x86_64-unknown-linux-gnu
1.31.0-x86_64-unknown-linux-gnu
1.32.0-x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.35.0 (3c235d560 2019-05-20)
```

**Comment by lotabout** [comment]
Confirmed. Very sorry for causing this trouble.
It's because I committed a `Cargo.lock` that depends on a wrong version of `tuikit`.

**Comment by lebensterben** [comment]
>Confirmed. Very sorry for causing this trouble. ...

@lotabout and @h-michael Thx  

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by lotabout** [comment]
Published a new version 0.6.8. Confirmed `cargo install skim` works locally.

**Comment by lebensterben** [comment]
>Published a new version 0.6.8. Confirmed `cargo install skim` works locally.

@lotabout Confirmed. Thanks.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
